### ISSUE TITLE
[EPIC] Add Alt Text to Existing Images in app

### ISSUE THREAD

**Issue by acovher2** [rokwire/illinois-app#5547]
### Description
_Please provide a detailed description of the epic._
_Example: As a (user or developer), I want to be able to (feature or function) so that (reason to implement)._

As a screen reader user, I want to be able to hear descriptions of images (alt text) so that I may also have the information they were intended to convey. 

### Issues
_Please link each related issue in a checklist below._

Alt text for images— this is the suggested phasing and priority order per Vanessa

**Group images***  #5549 

  - Group cover photo uploaded by admin
  - Group posts with an image

**Cover images in other features**  #5550 

  - My Courses detail panel default image*
  - Dining locations detail panel default images* 
  - MyMcKinley detail panel default images

**Event images**  #5553 

  - Images uploaded via the Illinois app
  - Images for Big 10 Athletic events
  - Images from Fusion for Campus Recreation events
  - Images from Webtools
  - Default image when there is no image provided
  - It's Game Day images

**Profile images in Groups**  #5548 

**Directory of Users**   #5554 

     (Note: if the user has disabled image from Directory this should skip as decorative)

_* higher usage_

**Comment by TodorBachvarov** [comment]
@acovher2, for images which are missing the Alt Text I'm implementing the same approach as in this comment: https://github.com/rokwire/illinois-app/issues/5548#issuecomment-3553049242. These images will be treated as decorative and will be skipped from the screen reader.
